src/ui-lib/sass/05-components/01-atoms/03-forms/_01-form-group.scss

```diff
@@ -0,0 +1,10 @@
+.grav-c-form-group {
+  * {
+    margin: 0;
+  }
+
+  > label {
+    display: block;
+    margin-bottom: 0.5em;
```

Comment:
Would it not be more succinct to simply override the `input`'s `margin-top`? As the first child of the group element, the `label` will already have a `margin-top: 0;`.

So, something like this:
```sass
.grav-c-form-group {
  > label {
    display: block;
  }

  > input {
    margin-top: $grav-sp-s;
  }
}
```